packages/apps/spaces/app/organization/[id]/src/components/Tabs/panels/AccountPanel/Contract/ContractSubtitle.tsx

```diff
@@ -70,8 +77,41 @@ export const ContractSubtitle = ({ data }: { data: Contract }) => {
     (e) => e.value === data?.contractRenewalCycle,
   )?.label;
 
+  if (
+    !renewalPeriod &&
+    !hasStartedService &&
+    !serviceStartDate &&
+    data?.contractRenewalCycle
+  ) {
+    return (
+      <Text>
```

Comment:
I think placing a button inside a Text which is a `p` element will throw a runtime error in the console about domNestingValidation or something similar. Please use a div to wrap. I've seen this error in a lot of places and they always come up when anything other than spans or raw text is passed as children to Text components.

Reply:
Also it would be nice to also use `span` or `p` instead of `Text. 🙏

Reply:
fixed, btn remove cause this whole section is clickable button